src/storhaug

```diff
@@ -291,29 +305,44 @@ create_virt_ip()
     local ipcount=${1}; shift
     local ip=${1}; shift
 
-    pcs -f ${cibfile} resource create vip${ipcount} ocf:heartbeat:IPaddr2 \
-        params \
-            ip=${ip} \
-            flush_routes="true" \
-        op monitor interval=60s \
-        meta resource-stickiness="0"
+    if [[ "X${HA_SVC_NFS}X" == "XyesX" ]]; then
+        pcs -f ${cibfile} resource create vip${ipcount} ocf:heartbeat:IPaddr2 \
+            --group group${ipcount} \
+            params \
+                ip=${ip} \
+                flush_routes="true" \
+            op monitor interval=60s \
+            meta resource-stickiness="0"
+
+        pcs -f ${cibfile} resource create vip${ipcount}_block ocf:heartbeat:portblock \
+            protocol=tcp portno=2049 action=block ip=${ip} --group group${ipcount}
+
+        pcs -f ${cibfile} resource create vip${ipcount}_unblock ocf:heartbeat:portblock \
+            protocol=tcp portno=2049 action=unblock ip=${ip} --group group${ipcount} \
+            --after vip${ipcount} reset_local_on_unblock_stop=true \
+            tickle_dir=${HA_VOL_MNT}/nfs-ganesha/tickle_dir/ \
+            op stop timeout=${PORTBLOCK_UNBLOCK_TIMEOUT} \
+            op start timeout=${PORTBLOCK_UNBLOCK_TIMEOUT} \
+            op monitor interval=10s timeout=${PORTBLOCK_UNBLOCK_TIMEOUT}
+    else
+        pcs -f ${cibfile} resource create vip${ipcount} ocf:heartbeat:IPaddr2 \
+            params \
+                ip=${ip} \
+                flush_routes="true" \
+            op monitor interval=60s \
+            meta resource-stickiness="0"
+    fi
 
     pcs -f ${cibfile} constraint location vip${ipcount} rule resource-discovery=exclusive score=0 role eq storage
 
     pcs -f ${cibfile} resource create vip${ipcount}_trigger ocf:heartbeat:ganesha_trigger \
-        --clone vip${ipcount}_trigger-clone vip${ipcount}_trigger \
-	meta interleave="true" globally-unique="false" notify="true" clone-max=1
-
-    pcs -f ${cibfile} constraint colocation add vip${ipcount}_trigger-clone with vip${ipcount} INFINITY
-    pcs -f ${cibfile} constraint order vip${ipcount} then vip${ipcount}_trigger-clone
-
-    pcs -f ${cibfile} constraint colocation add vip${ipcount} with nfs-ganesha-clone INFINITY
```

Comment:
Why are you getting rid of the service constraints? A vip shouldn't start on a node that doesn't have the configured services running.